### ISSUE TITLE
Any example with React Implementation

### ISSUE THREAD

**Issue by Stancobridge** [jordankaerim/inertia-node#3]
I have been trying to get this work with react 17.0.2, but I can't get it to work
I created the react with CRA

It will be good if I can get any working example with react to know what am missing

**Comment by hashsaksham** [comment]
Yes same here, Need help in implementing this in React with Express.
Please Helpl

**Comment by manshu** [comment]
Same here. It seems to have an issue installing.
